docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Minishift failed to start because of ```Error: could not create vmnet interface, permission denied or no entitlement```.
+
+    The problem is likely to be that the xhyve driver is not able to clean up vmnet when a VM is removed.
+
```

Comment:
@Preeticp `vmnet.framework` is API which allow virtual machines to read and write packets and `vmnet` is interface which created using this API so here vmnet is right because that interface is not deleted.